examples/basic-routing/README.md

```diff
@@ -12,47 +12,6 @@ The code in [main.tf] creates a Route53 Zone and Records for the main domain and
 
 The `www.` subdomains are implement through CNAMES and point on the A-Records.
 
-```hcl
-module "route53" {
-  source  = "mineiros-io/route53/aws"
-  version = "~> 0.5.0"
-
-  name = "a-dev-mineiros.io"
-
-  records = [
```

Comment:
why remove this?

Reply:
There is 0 value in duplicating code among the `README.md` and the `main.tf` files, it just increases complexity. In newer modules we got rid of that habit also.